src/routes/api/users.js

```diff
@@ -13,11 +12,9 @@ import { roleIds } from '../../helpers/default';
 import userController from '../../controllers/userController';
 import indexController from '../../controllers/indexController';
 import validate from '../../middlewares/validate';
-import { findByEmail } from '../../utils/searchDb';
-
 
```

Comment:
You haven't removed all duplicate routes. Please check again.

Reply:
Alright, Fixed